app/models/account.rb

```diff
@@ -277,6 +282,24 @@ def config
   scope :created_before, ->(date) { where(['created_at <= ?', date]) }
   scope :created_after,  ->(date) { where(['created_at >= ?', date]) }
 
+  def self.should_be_automatically_suspended
```

Comment:
consider moving this code to `app/queries`.

Reply:
That sounds like a good idea 👍 I will check about it 👍